### ISSUE TITLE
1.3.4 via Xcode and SPM fails to build on macOS

### ISSUE THREAD

**Issue by gregcotten** [krzyzanowskim/CryptoSwift#838]
Running macOS 10.15.7 with Xcode 12.2 (12B45b).

Making a new Xcode project (macOS App) and adding the CryptoSwift swift package (1.3.4) causes a compile time error with code signing:

```
CodeSign [Redacted]/Library/Developer/Xcode/DerivedData/MyGreatTest-fiuanbbuztfnoobalglyfktpwnzs/Build/Products/Debug/MyGreatTest.app (in target 'MyGreatTest' from project 'MyGreatTest')
    cd [Redacted]/Desktop/MyGreatTest
    export CODESIGN_ALLOCATE\=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/codesign_allocate
    
Signing Identity:     "[Redacted]"

    /usr/bin/codesign --force --sign E99CEDF6505E90C9CADAE198ED1685665E486D54 -o runtime --entitlements [Redacted]/Library/Developer/Xcode/DerivedData/MyGreatTest-fiuanbbuztfnoobalglyfktpwnzs/Build/Intermediates.noindex/MyGreatTest.build/Debug/MyGreatTest.build/MyGreatTest.app.xcent --timestamp\=none [Redacted]/Library/Developer/Xcode/DerivedData/MyGreatTest-fiuanbbuztfnoobalglyfktpwnzs/Build/Products/Debug/MyGreatTest.app

[Redacted]/Library/Developer/Xcode/DerivedData/MyGreatTest-fiuanbbuztfnoobalglyfktpwnzs/Build/Products/Debug/MyGreatTest.app: code object is not signed at all
In subcomponent: [Redacted]/Library/Developer/Xcode/DerivedData/MyGreatTest-fiuanbbuztfnoobalglyfktpwnzs/Build/Products/Debug/MyGreatTest.app/Contents/Frameworks/CryptoSwift.framework
Command CodeSign failed with a nonzero exit code
```

No compile issues with 1.3.3. I imagine this has something to do with the binary XCFramework distribution for macOS added in 1.3.4?

**Comment by krzyzanowskim** [comment]
uhm... what again with the Xcode. How did you integrate, is it SPM?

**Comment by gregcotten** [comment]
Yes, SPM!

**Comment by krzyzanowskim** [comment]
smthg smtng Xcode signing somehwere. Any ideas on how to fix it?

**Comment by gregcotten** [comment]
Unclear at the moment, but when I get some time I will investigate further.

**Comment by krzyzanowskim** [comment]
fixed with 1.3.5 release (hopefully). Please let me know if that worked out as expected.

**Comment by gregcotten** [comment]
yep, definitely fixed!
